themes/default/theme/src/ts/docs-main.ts

```diff
@@ -0,0 +1,83 @@
+var docsMainNavToggleWrapper = $(".docs-main-nav-toggle-wrapper");
```

Comment:
we typically always use `const` or `let` to declare variables. Using `var` is generally considered bad practice in javascript and typescript at this point and is just something that has been left around in the language since its early days. It is mainly because `var` variables do not respect block scoping and easily can create unwanted behavior if not used carefully. You can change all the vars here to let and it should continue to function as is.